src/+groundwater/calculateGroundwaterRecharge.m

```diff
@@ -103,13 +103,14 @@ soilThick               cumulative soil layers thickness (from top to bottom)
         warning('Index of the bottom layer boundary that includes groundwater head ~= index of the recharge layer + 1!');
     end
 
-    % Outputs to be exported in csv
+    % Outputs to be exported in csv or exposed to BMI
     gwfluxes.QLh = QLh_flip(indxRchrg);
     gwfluxes.QLT = QLT_flip(indxRchrg);
     gwfluxes.QLa = QLa_flip(indxRchrg);
     gwfluxes.QVH = QVH_flip(indxRchrg);
     gwfluxes.QVT = QVT_flip(indxRchrg);
     gwfluxes.QVa = QVa_flip(indxRchrg);
+    gwfluxes.indxRchrg = indxRchrg;
```

Comment:
The function `groundwater.calculateGroundwaterRecharge` is only called if Groundwater coupling is enabled, see [here](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/1f3f02d88d2e9a69a50f56f73135e01cd26f9df4/src/STEMMUS_SCOPE.m#L779-L780). If coupling is not enabled, what would be the value of `gwfluxes.indxRchrg`, `zero` or `None`? the variable `gwfluxes.indxRchrg` should be added in [this else block](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/1f3f02d88d2e9a69a50f56f73135e01cd26f9df4/src/STEMMUS_SCOPE.m#L785).

Reply:
> The function `groundwater.calculateGroundwaterRecharge` is only called if Groundwater coupling is enabled, see [here](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/1f3f02d88d2e9a69a50f56f73135e01cd26f9df4/src/STEMMUS_SCOPE.m#L779-L780). If coupling is not enabled, what would be the value of `gwfluxes.indxRchrg`, `zero` or `None`? the variable `gwfluxes.indxRchrg` should be added in [this else block](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/1f3f02d88d2e9a69a50f56f73135e01cd26f9df4/src/STEMMUS_SCOPE.m#L785).

Okay, added [here](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/4961584424a56ac966fac9d88ab3b1c83b94b795/src/STEMMUS_SCOPE.m#L786C1-L786C38)